lib/memcached.js

```diff
@@ -913,7 +913,7 @@ Client.config = {
     value = Utils.escapeValue(value);
 
     length = Buffer.byteLength(value);
-    if (length > this.maxValue) {
+    if (length + fullkey.length + 71 > this.maxValue) {
```

Comment:
Thank you for the tips. We use it with both memcached and couchbase ourselves. I will do a bit more research and tweak the code accordingly.
Thanks!